plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -428,7 +445,7 @@ fu_thunderbolt_device_init(FuThunderboltDevice *self)
 	fu_device_add_icon(FU_DEVICE(self), "thunderbolt");
 	fu_device_add_protocol(FU_DEVICE(self), "com.intel.thunderbolt");
 	fu_device_add_internal_flag(FU_DEVICE(self), FU_DEVICE_INTERNAL_FLAG_NO_PROBE_COMPLETE);
-	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_PAIR);
+	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_BCD);
```

Comment:
this will change both the retimer and the controller.  Is that intended?

Reply:
hmm... the versioning convention apply only the thunderbolt controller, the verfmt needs to be set somewhere else.